Framework/Source/Graphics/Model/Loaders/AssimpModelImporter.cpp

```diff
@@ -815,19 +818,14 @@ namespace Falcor
         // Must have position!!!
         if (pAiMesh->HasPositions() == false)
         {
-            logError("Loaded mesh with no positions!");
+            logError("AssimpModelImporter: Loaded mesh with no positions!");
             return nullptr;
         }
 
-        if (pAiMesh->GetNumUVChannels() > 1)
+        // Must at least have texture UV channel
+        if ((pAiMesh->HasTextureCoords(0) == false))
```

Comment:
This is incorrect. The previous code made sure we have a single UV channel at best. You want to make sure we at at most 2 UV sets